UNRELEASED.md

```diff
@@ -34,6 +34,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 - Bump react-utilites to remove a transitive dependency on core-js. ([#1343](https://github.com/Shopify/polaris-react/pull/1343))
 
+- Updated App Bridge to version 1.3.3 ([#1349](https://github.com/Shopify/polaris-react/pull/1349))
```

Comment:
Typo here, should be `1.3.0`

Reply:
My bad! I'll put up another pr to fix it